biztalk/technical-guides/pre-configuration-database-optimizations1.md

```diff
@@ -20,7 +20,7 @@ BizTalk Server is an extremely database-intensive application that may require t
 
 ## Set NTFS File Allocation Unit
 
- SQL Server stores its data in **Extents**, which are groups of eight 8K pages. Therefore, to optimize disk performance, set the NTFS Allocation Unit size to 64KB as described in the “Disk Configuration Best Practices” section of the SQL Server best practices article “Predeployment I/O Best Practices” available at [https://go.microsoft.com/fwlink/?LinkId=140818](/previous-versions/sql/sql-server-2005/administrator/cc966412(v=technet.10)). For more information about SQL Server pages and extents see the SQL Server Books Online topic [Understanding Pages and Extents](/previous-versions/sql/sql-server-2008-r2/ms190969(v=sql.105)) ( HYPERLINK "<https://go.microsoft.com/fwlink/?LinkId=148939>" <https://go.microsoft.com/fwlink/?LinkId=148939>).
+ SQL Server stores its data in **Extents**, which are groups of eight 8K pages. Therefore, to optimize disk performance, set the NTFS Allocation Unit size to 64KB as described in the “Disk Configuration Best Practices” section of the SQL Server best practices article [Predeployment I/O Best Practices](https://go.microsoft.com/fwlink/?LinkId=140818). For more information about SQL Server pages and extents see the SQL Server Books Online topic [Understanding Pages and Extents](/previous-versions/sql/sql-server-2008-r2/ms190969(v=sql.105)) ( HYPERLINK "<https://go.microsoft.com/fwlink/?LinkId=148939>" <https://go.microsoft.com/fwlink/?LinkId=148939>).
 
 ## Database planning considerations
  We recommend that you host your [!INCLUDE[btsSQLServerNoVersion](../includes/btssqlservernoversion-md.md)] databases on fast storage (for example, fast SAN disks or fast SCSI disks). We recommend RAID 10 (1+0) instead of RAID 5 because raid 5 is slower at writing. Newer SAN disks have very large memory caches, so in these cases, the RAID selection is not as important. To increase performance, databases and their log files can reside on different physical disks.
```

Comment:
```suggestion
Microsoft recommends that you host your [!INCLUDE[btsSQLServerNoVersion](../includes/btssqlservernoversion-md.md)] databases on fast storage (for example, fast SAN disks or fast SCSI disks). Microsoft recommends RAID 10 (1+0) instead of RAID 5 because RAID 5 is slower at writing. Newer SAN disks have very large memory caches, so in these cases, the RAID selection is not as important. To increase performance, databases and their log files can reside on different physical disks.
```